pkg/topology/arangodb/arangodb-peer-change.go

```diff
@@ -0,0 +1,20 @@
+package arangodb
+
+func (m *arangoDB) peerChangeHandler(obj *message.PeerStateChange) {
+	glog.Infof("><SB> peer change handler - update db with peer change message data")
+
+	// creates a new BGP peer document in Arango's BGPPeer vertex collection
+	bgp_peer_document := &arangoDB.BGPPeer{
+		RouterID:	message.LocalBGPID
+		LocalIP:	message.LocalIP
+		LocalASN:	message.LocalASN
+		PeerIP:		message.RemoteIP
+		PeerBGPID:	message.RemoteBGPID
+		PeerASN:	message.RemoteASN
+	}
+	if err := m.arangoDB.Upsert(bgp_peer_document); err != nil {
```

Comment:
If Upsert is methof provided by arango go client, then it is not yet available as I still need to add arango client instantiation.  but if this method is/will be developed by you, then you would call it **a.Upsert**